bip-MarcoPon-01.mediawiki

```diff
@@ -0,0 +1,67 @@
+<pre>
+  BIP: MarcoPon-01
+  Title: URI scheme for Blockchain references / exploration
+  Author: Marco Pontello <marcopon@gmail.com>
+  Status: Draft
+  Type: Standards Track
+  Created: 29 August 2015
+</pre>
+
+==Abstract==
+
+This BIP propose an URI scheme for looking up blocks, transactions, addresses on a Blockchain explorer, or in general to make proper Blockchain references
+
+==Motivation==
+
+The purpose of this URI scheme is to enable users to handle all the requests for details about blocks, transactions, etc. with their preferred tool (being that a web service or a local application).
+Currently a Bitcoin client usually point to an arbitrary blockchain explorer when the user look for the details of a transaction (es. Bitcoin Wallet use BitEasy, Mycelium or Electrum use Blockchain.info, etc.).
+Other times resorting to cut&paste is needed.
+The same happens with posts and messages that reference some particular txs or blocks, if they provide links at all.
+
+==Specification==
+
+The URI follow this simple form:
+
+ <nowiki>blockchain:[//chain]</type/hash></nowiki>
+
+Where:
+
+;chain:
+: (optional) to uniquely point to a specific chain, the hash of the corresponding genesis block is used (see [http://https://github.com/jtimon/bitcoin/commit/3191d5e8e75687a27cf466b7a4c70bdc04809d39 Bitcoin src/chainparams.cpp]). In principle some kind of alias/mnemonic could also be used, but that is out of the scope of this BIP, and maybe could be developed in another subsequent one.
```

Comment:
Also the link seems broken. An updated version of that commit is contained in #6382, in https://github.com/jtimon/bitcoin/commit/599019a9127abb1ada30eaf2c1e7b38e5d4239ec (but that commit will need to be rebased again as 6382's sub-PRs get merged/closed). It was once opened independently as #6230 but it is closed for now.